index.html

```diff
@@ -0,0 +1,161 @@
+ <DOCKTYPE html>
+ <html>
+    <head>
+        <meta charset="utf-8">
+        <title> Барбершоп "Бородинский"</title>
+    </head>
+    <body>
+        <header class="header">
+            <nav class="main-navigation">
+                    <a class="main-header-logo">
+                        <img src="img/index-logo.svg" width="368" height="153" alt="Логотип барбершопа Бородинский">
+                    </a>
+                    <ul class="site-navigation">
+                        <li><a href="#">Информация</a></li>
+                        <li><a href="#">Новости</a></li>
+                        <li><a href="#">Прайс-лист</a></li>
+                        <li><a href="#">Магазин</a></li>
+                        <li><a href="#">Контакты</a></li>
+                    </ul>
+                    <ul class="user-navigation">
+                        <li class="login-link">
+                            <a class="login-link" href="login.html">Вход</a>
+                        </li>
+                    </ul>
+            </nav>
+        </header>
+        <main class="container">
+            <h1 class="visually-hidden">Барбершоп Бородинский"</h1>
+            <section class="features">
+                <h2 class="visually-hidden">Преимущества</h2>
+                    <ul class="features-list">
+                        <li class="feature-item">
```

Comment:
```suggestion
                        <li class="features-item">
```

Если это item, то вписываешь название блока, к которому item относится